sdks/java/extensions/sorter/README.md

```diff
@@ -18,11 +18,11 @@
 -->
 
 #Sorter
-This module provides the SortValues transform, which takes a `PCollection<KV<K,Iterable<V>>> and produces a PCollection<KV<K,Iterable<V>>>` where the `Iterable<V>` has been sorted for each key. It will efficiently and scalably sort the iterables, even if they are large (do not fit in memory).
+This module provides the SortValues transform, which takes a `PCollection<KV<K, Iterable<KV<K2, V>>>>` and produces a `PCollection<KV<K, Iterable<KV<K2, V>>>>` where the `Iterable<<KV<K2, V>>>` has been sorted for each secondary key (`K2`). It will efficiently and scalably sort the iterables, even if they are large (do not fit in memory).
```

Comment:
Good catch on angle braces. Changed wording to make it more clear as suggested. Byte encoding of V is not used as a tie-breaker.